CriticalMass/it.lproj/Localizable.strings

```diff
@@ -30,49 +30,61 @@
 "rules.source" = "Immagini di criticalmass-hh.de (CC BY-NC-ND 3.0 DE)";
 
 /* No comment provided by engineer. */
-"rules.text.brake" = "Evita di fermarti all'improvviso. Se non c'è altro modo, cerca di mettere in guardia chi ti sta dietro con un gesto delle mani.";
+"rules.title" = "Regole";
 
 /* No comment provided by engineer. */
-"rules.text.contraflow" = "Fai attenzione a non guidare contromano!";
+"rules.title.communication" = "Communication";
 
 /* No comment provided by engineer. */
-"rules.text.cork" = "Proteggi i motociclisti da se stessi, creando un ingorgo!\n\nPer mantenere la coesione del gruppo, blocca il traffico dalle strade laterali per fare in modo che la massa possa passare i semafori rossi senza interruzioni.";
+"rules.title.vibes" = "Good Vibes";
 
 /* No comment provided by engineer. */
-"rules.text.gently" = "Quando sei in testa, non correre.\n\nSe invece sei dietro, chiudi i buchi!";
+"rules.title.contraflow" = "Oncoming Traffic";
 
 /* No comment provided by engineer. */
-"rules.text.green" = "Se sei in testa al gruppo ed incontri un semaforo rosso, devi attendere che diventi verde.";
+"rules.title.pace" = "Pace Yourself";
 
 /* No comment provided by engineer. */
-"rules.text.haveFun" = "Goditi le strade senza auto. Dai un'occhiata alle Sound Bikes. Scambia quattro chiacchere con automobilisti e pedoni, spiegando cosa sta succendendo.\n\nE la cosa più importante: divertiti!";
+"rules.title.mass" = "Unified Mass";
 
 /* No comment provided by engineer. */
-"rules.text.stayLoose" = "Non farti provocare. Sii amichevole con la polizia, gli automobilisti e chiunque altro, anche nel caso loro non lo fossero.";
+"rules.title.emergency" = "Police Escort";
```

Comment:
also "Emergency Vehicle" (for the moment)

Reply:
Emergency Vehicle could be "Veicoli d'Emergenza"